toolshed/routes/frontend/frontend.index.ts

```diff
@@ -3,9 +3,18 @@ import * as path from "@std/path";
 import { createRouter } from "@/lib/create-app.ts";
 import { applyProxy } from "./frontend.proxy.ts";
 import { applyStatic } from "./frontend.static.ts";
+import { cors } from "@hono/hono/cors";
 
 const router = createRouter();
 
+router.use(
+  "*",
+  cors({
```

Comment:
```suggestion
  // setup CORS otherwise `bootstrap.js` module import from the srcdoc iframe
  // is blocked and charm fails to bootstrap
  cors({
```

Reply:
@jsantell how's that ☝️ ?